docs/tutorial/electron-timelines.md

```diff
@@ -1,23 +1,26 @@
 # Electron Release Timelines
 
+Special notes:
```

Comment:
I _think_ that error message means the linter is asking for a blank line between this and the list that follows

>  docs/tutorial/electron-timelines.md:4 MD032/blanks-around-lists Lists should be surrounded by blank lines [Context: "* The `-beta.1` and `stable` d..."]

Reply:
Oops. Let me fix